drivers/mil_blueview_driver/include/blueview_ros_driver.hpp

```diff
@@ -15,18 +15,19 @@ class BlueViewRosDriver
 public:
   BlueViewRosDriver();
   void initParams();
-
+  double period_seconds;
```

Comment:
This should be replaced with getter, setter funcs (if necessary), bad practice to have public data members.